### PR TITLE
docs: document buffer limit for client

### LINKED ISSUE #3109: docs: make `per_connection_buffer_limit_bytes` configurable
*Description*:

relates to https://github.com/envoyproxy/gateway/issues/2600

### PR DESCRIPTION
**Pull request by yaelSchechter**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #3109

### CODE REVIEW

**Review by zirain — DISMISSED** [review]

File: site/content/en/latest/tasks/traffic/client-traffic-policy.md [hunk 1]
```diff
@@ -573,5 +573,26 @@ curl -v http://$GATEWAY_HOST/get \
 request timeout
 ```
 
+### Configure Downstream Per Connection Buffer Limit
+
+This feature allows you to set a soft limit on size of the listener’s new connection read and write buffers.
+The size is configured using the `resource.Quantity` format, see examples [here](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory).
+
+```shell
```
Comment (shawnh2) [inline site/content/en/latest/tasks/traffic/client-traffic-policy.md:581]: we should also make this with two tabs using tabpane like L501, one is `Apply from stdin`, and the other one is `Apply from file`.

Comment (yaelSchechter) [reply site/content/en/latest/tasks/traffic/client-traffic-policy.md:584]: thanks, fixed both

**Review by shawnh2 — APPROVED** [review]
LGTM thanks!

File: site/content/en/v1.0.1/tasks/traffic/client-traffic-policy.md [hunk 2]
```diff
@@ -412,7 +412,7 @@ Handling connection for 8888
```
Comment (guydc) [inline site/content/en/v1.0.1/tasks/traffic/client-traffic-policy.md:412]: Is this feature available in 1.0.1 ? I only see the commit in latest: https://github.com/envoyproxy/gateway/commit/6941624e1fbe2e3c2f9d8ef47b71862cc4a53749

Comment (yaelSchechter) [reply site/content/en/v1.0.1/tasks/traffic/client-traffic-policy.md:412]: removed, thanks

**Comment by guydc** [comment]
@yaelSchechter - can you resolve the conflicts?

**Review by guydc — APPROVED** [review]

**Pull request merged**